ci-operator/README.md

```diff
@@ -0,0 +1,154 @@
+ci-operator
+===========
+
+
+End-to-end tests
+----------------
+
+The preferred way to write e2e tests is using `ci-operator`.  See the
+documentation for details on how to download, build, and execute it:
+
+https://github.com/openshift/ci-operator.git
+
+The first step is writing a configuration file for `ci-operator` for the
+repository.  Many projects already have one, which takes care of most of the CI
```

Comment:
"projects" --> "OpenShift component repositories" is more accurate? "project" is an overloaded term, best to be specific.

Reply:
"already have one" --> suggest linking to an illustrative real-world example.

Reply:
Changed.